tests/func/test_remote.py

```diff
@@ -169,3 +170,16 @@ def test(self):
 
         ret = main(["push", "-r", self.upper_case_remote_name])
         self.assertEqual(ret, 0)
+
+
+def test_large_dir_progress(repo_dir, dvc):
+    from dvc.utils import LARGE_DIR_SIZE
+    from dvc.progress import progress
+
+    # Create a "large dir"
+    for i in range(LARGE_DIR_SIZE + 1):
+        repo_dir.create("gen/{}.txt".format(i), str(i))
```

Comment:
Please use os.path.join. I know that in this case python is able to handle posix relpath even on windows correctly, but we try to generally use join instead to prevent potential errors in the future.